.github/workflows/tests.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       matrix:
         os: [macos-latest, ubuntu-latest, windows-latest]
-        python-version: [3.6, 3.9, 3.9.5]
+        python-version: [3.6, 3.9, 3.9.6]
```

Comment:
Why are 3.7 and 3.8 skipped?

Reply:
We should be testing for those verisons. I'm currenly currently using the python version i have installed and that github actions supports. so i will add the support verisons thanks for this